Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3619,7 +3619,7 @@ public void run() {
 	 * @see BranchError	
 	 */
 	public interface BranchListResponseListener {
-		public void onReceivingResponse(JSONArray list, BranchError error);
+		void onReceivingResponse(JSONArray list, BranchError error);
```

Comment:
I generally dislike this convention. However, it appears that most style guides do recommend it.